### ISSUE TITLE
Verilog Header files support

### ISSUE THREAD

**Issue by alessandrothea** [ipbus/ipbb#159]
Add `.vh` among the file types known to ipbb

**Comment by tj710** [comment]
In fact, it does not work using the build.tcl and settings.tcl scripts referenced in the deep file, for example, because the files are not yet added to the project. Is there an easy way to run a tcl script with ipbb after "proj create" has finished? Thanks!

**Comment by tj710** [comment]
closing and will open a new issue for corresponding source file attribute

**Comment by alessandrothea** [comment]
Yes, there's a way: the `-f/--finalize` `setup` command flag, i.e. `setup -f abc.tcl` will execute `abc.tcl` after all sources have been imported.

**Comment by tj710** [comment]
hmm, I still have issues e.g
I put in my dep file:
setup --finalise ../cfg presynth.tcl
where presynth.tcl contains "get_files -all"
but at ipbb vivado project, when the file is sources, it reports:

generate-project | # get_files -all
generate-project | WARNING: [Vivado 12-818] No files matched '*'

Perhaps I am doing something else wrong.
